src/components/Common/MailingListSubscribeForm.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <div>
+    <div id="mc_embed_signup">
+      <form
+        action="https://sharestake.us1.list-manage.com/subscribe/post?u=6cb6b636781c4fcc0b7cbccf1&amp;id=1e3ebab5ba"
```

Comment:
Thanks for catching that. I fixed it. Now it's "https://sharedstake.us1.list-manage.com/subscribe/post?u=6cb6b636781c4fcc0b7cbccf1&amp;id=1e3ebab5ba"